mangaki/mangaki/utils/knn.py

```diff
@@ -118,20 +91,32 @@ def fit(self, X=None, y=None, whole_dataset=False):
             self.M[user_id, work_id] = rating
         for work_id in self.nb_ratings:
             self.mean_score[work_id] = self.sum_ratings[work_id] / self.nb_ratings[work_id]
-        self.get_all_neighbors()
+        if not self.single_user:
```

Comment:
Cf oob discution; let's remove `single_user` and move the call to `get_neighbors` to `predict`.